lib/core/versions/latest/CreateOperation.ts

```diff
@@ -33,53 +30,29 @@ export default class CreateOperation implements OperationModel {
   public readonly delta: DeltaModel | undefined;
 
   /** Encoded string of the suffix data. */
-  public readonly encodedSuffixData: string;
-
-  /** Encoded string of the delta. */
-  public readonly encodedDelta: string | undefined;
 
   /**
    * NOTE: should only be used by `parse()` and `parseObject()` else the constructed instance could be invalid.
    */
   private constructor (
     operationBuffer: Buffer,
     didUniqueSuffix: string,
-    encodedSuffixData: string,
     suffixData: SuffixDataModel,
-    encodedDelta: string | undefined,
     delta: DeltaModel | undefined) {
     this.didUniqueSuffix = didUniqueSuffix;
     this.type = OperationType.Create;
     this.operationBuffer = operationBuffer;
-    this.encodedSuffixData = encodedSuffixData;
     this.suffixData = suffixData;
-    this.encodedDelta = encodedDelta;
     this.delta = delta;
   }
 
-  /**
-   * Computes the DID unique suffix given the encoded suffix data string.
-   */
-  private static computeDidUniqueSuffix (encodedSuffixData: string): string {
-    const suffixDataBuffer = Encoder.decodeAsBuffer(encodedSuffixData);
-    const multihash = Multihash.hash(suffixDataBuffer);
-    const encodedMultihash = Encoder.encode(multihash);
-    return encodedMultihash;
-  }
-
   /**
    * Parses the given buffer as a `CreateOperation`.
    */
   public static async parse (operationBuffer: Buffer): Promise<CreateOperation> {
     const operationJsonString = operationBuffer.toString();
     const operationObject = await JsonAsync.parse(operationJsonString);
-    let createOperation;
-    if (typeof operationObject.suffixData === 'string') {
-      // TODO: SIP 2 #781 deprecates this. Should be deleted when fully switched over
-      createOperation = await CreateOperation.parseObject(operationObject, operationBuffer, false);
-    } else {
-      createOperation = CreateOperation.parseJcsObject(operationObject, operationBuffer);
-    }
+    const createOperation = CreateOperation.parseJcsObject(operationObject, operationBuffer);
```

Comment:
Nit: consider renaming back to `parseObject` to be consistent with other operation classes.